### PR TITLE
Added logo and icons in the header

### LINKED ISSUE #75: Adding clickable Edam logo
How about adding clickable Edam logo 

 
Something like this:
This format looks better to me than the present format.
Screenshot_2021-04-03-09-38-40-36

IMG_20210403_085632

In the header at the place of Edam and shifting Edam ontology rightward in accordance, that logo will link to [edamontology] (www.edamontology.org) so link in the red box beside Edam explanation can be removed.

Just for beautification purpose I can say :-P

All the websites have their logos displayed at the front page at least so I proposed this. And I couldn't find Edam logo on the Edam browser

### PR DESCRIPTION
**Pull request by raashika03**
### Checklist

- [x] I indicated which issue (if any) is closed with this PR using [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] I only changed lines related to my PR (no bulk reformating)
- [x] I indicated the source and check the license if I re-use code, or I did not re-use code
- [x] I made my best to solve only one issue in this PR, or explain why multi had to be solved at once.

### Issue

resolve #75 

### Details

Before:
Screenshot from 2021-04-26 23-19-46
After:
Screenshot from 2021-04-26 23-17-58
Screenshot from 2021-04-26 23-18-08

@bryan-brancotte This's the initial version Please suggest changes for like what should I write beside github/gitter icon instead of edam-browser.

### CODE REVIEW

**Comment by raashika03** [comment]
I'll add more commits when you'll clarify your thoughts as I asked [here](https://github.com/edamontology/edam-browser/issues/75#issuecomment-826885204). For now, this is the initial version.

**Comment by bryan-brancotte** [comment]
Hi @raashika03 

I like the look, although I think we could save space by moving the button like this:
116127778-d8cd8000-a6e5-11eb-8fe6-beaeb23796df


My main concerne is regarding its implementation, I read the css and html and you kept the `col-xs-12 ...` will  adding `id="..."` along with `position:absolute`. Mixing both of them is not a good practice, `col-xs-12` and alike are meant to be relative, so fixing it or its descendant to absolute can only end up with complicated css code and unexepected behavior.

Although I think good results could be achieve with position absolute, I recommend you to try using `dispay:flex` (here are some [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox)). It will allow you to cut the space, automatically assign the blank space to one or another.

Here is an example of how the area should be cut, when using display:flex
cut
Refinement could done also to keep the button and the search bar in the vertical middle of the global header area.

**Comment by raashika03** [comment]
Oki @bryan-brancotte I understood well thank you! I'll implement it this way.

**Comment by raashika03** [comment]
@bryan-brancotte done! check once please!

**Comment by raashika03** [comment]
@bryan-brancotte it's done! Please summarize your views.
Screenshot from 2021-04-27 03-03-35

File: index.html [hunk 1]
```diff
@@ -129,9 +146,6 @@ <h1 class="no-icon text-center">
                 </div>
             </h1>
         </div>
-        <div class="col-xs-12">
-            <h4>EDAM is a simple ontology of well established, familiar concepts that are prevalent within
-                bioinformatics [<a target="_blank" href="http://edamontology.org/">edamontology.org</a>]</h4>
         </div>
```
Comment (bryan-brancotte) [inline index.html:149]: should also have been removed

File: index.html [hunk 2]
```diff
@@ -47,8 +47,25 @@
...
 <div class="container-fluid">
     <div class="page-header row">
-        <div class="col-xs-12 col-sm-6 col-lg-3">
+        <div class="col-xs-12 col-sm-6 col-lg-3 flex-container" id="header-desc">
+            <img src="https://github.com/edamontology/edamontology/raw/main/EDAMrelations.png" alt="EDAM Logo"/>
+            <div>
             <h1 id="pageTitle">EDAM ontology</h1>
+            <p class="edam-defn">EDAM is a simple ontology of well established, familiar concepts that are prevalent within bioinformatics.</p>
+            <div class="header-icon">
+                <p>
```
Comment (bryan-brancotte) [inline index.html:56]: using `<p>` with `display: inline-block;` is the same as using span, so why no using it ? (and adding the margin in the css?)

**Review by bryan-brancotte — CHANGES_REQUESTED** [review]
Hi, Here are some minor changes but you can go more in the usage of display flex. I'll soon do a PR on your branch so you can see the difference, adapte/accept.

**Comment by raashika03** [comment]
Actually I think these div opening and closing are handled I moved that EDAM/ custom inside that flex-container class so only change in opening tag is showing. 
I checked its working after final commit. 
It's working sound. Correct me if i'm wrong.

File: index.html [hunk 3]
```diff
@@ -47,8 +47,25 @@
 <body>
 <div class="container-fluid">
     <div class="page-header row">
-        <div class="col-xs-12 col-sm-6 col-lg-3">
+        <div class="col-xs-12 col-sm-6 col-lg-3 flex-container" id="header-desc">
+            <img src="https://github.com/edamontology/edamontology/raw/main/EDAMrelations.png" alt="EDAM Logo"/>
+            <div>
```
[the remark this answers is not in the document]
Comment (bryan-brancotte) [reply index.html:52]: @raashika03 as this div is not closed for now it cause the second (buttons) and third (search bar) block to be in the first (logo&title). Check the PR I made to your branche, you can git clone to see how it works, do not forget to refresh cache

**Comment by raashika03** [comment]
Screenshot from 2021-04-27 19-55-11
So this's final look @bryan-brancotte 
And I apologise for those indentation, will keep in mind from next time :sweat_smile:

**Comment by raashika03** [comment]
But now header kind of design is removed. I kept that knowingly and aligned space bar and that like this to prevent capturing more space.
But, this also looks good. though spacing seems more.

**Comment by bryan-brancotte** [comment]
The screenshot is not what I get (even before my two quick commit), maybe refreshing the css (ctrl shift F5) ?

**Comment by raashika03** [comment]
There's no cache issue I checked.

**Comment by raashika03** [comment]
I think I made some mistake while reverting changes.

**Comment by raashika03** [comment]
Screenshot from 2021-04-27 21-58-57
This's the final result. I messed up locally (checked by deleting the previous repo locally and changing the code in this PR in that local new repo). Code is sound to be merged. @bryan-brancotte

**Comment by raashika03** [comment]
Twitter logo is there at the bottom. @bryan-brancotte

**Review by bryan-brancotte — APPROVED** [review]
We are good to merge

**Comment by bryan-brancotte** [comment]
> Twitter logo is there at the bottom. @bryan-brancotte

Yes, but I had the feeling that we were adding what was formerly at the bottom at the top, I was about to create an issue on whether we should now delete them from the bottom as they are in the header

**Comment by raashika03** [comment]
No but at the the bottom all of them points to edamontology not to Edam browser.

**Comment by raashika03** [comment]
Can you check [this] once. 
A proposal regarding the footer has been covered up. 
So removal might not be needed.

**Comment by bryan-brancotte** [comment]
I missed #77, ok so no need for a new issue as the question is already raised there.

I though the gitter was also edam-browser in the bottom. As edam-browser does not have a twitter account, pointing to edamontology twitter is not so invalide.

I thus propose we keep the twitter top and bottom, leave the footer as is a, and let #77 take care of it.

P.S: once https://github.com/edamontology/edamontology/pull/674 is merge, logo will magically re-appear.

**Comment by bryan-brancotte** [comment]
so ? merging ?

**Comment by raashika03** [comment]
Yes we're good to merge.

**Pull request merged**